### ISSUE TITLE
Use ROS drive to recieve line measurements

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#23]
## User story
1. As a SD
2. I want / need to use the provided package
3. So that we can recieve the line measurements in ROS

https://github.com/SICKAG/sick_lidar_localization

## Acceptance criteria
* Recieve line measurements in ROS

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by UmangBR** [comment]
The LiDAR-LOC software has a virtual track sensor. The signals of the virtual track sensor are sent out via UDP. With the ROS driver for LiDAR-LOC the UDP packets can be received and published as ROS Topic. The ROS driver is already unpacked and compiled on the PC in the ROS Workspace.  With the following instructions the driver can be started https://github.com/SICKAG/sick_lidar_localization/blob/master/README.md#run-sick_lidar_localization
What still has to be done is
1. Configure the virtual sensor in LiDAR-LOC (#22)
2. Adjust the parameters of the ROS driver (if necessary) to receive the UDP packet
3. Start the driver and test if the virtual track sensor signals arrive.
